services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/extensions/AtomicCounterExtension.java

```diff
@@ -37,10 +39,11 @@
 import software.amazon.awssdk.enhanced.dynamodb.update.UpdateExpression;
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
 import software.amazon.awssdk.utils.CollectionUtils;
+import software.amazon.awssdk.utils.Logger;
 
 /**
- * This extension enables atomic counter attributes to be written to the database.
- * The extension is loaded by default when you instantiate a
+ * This extension enables atomic counter attributes to be changed in DynamoDb by creating instructions for modifying
```

Comment:
Can we update the Javadoc to mention that users updating the fields marked with atomic counter manually has no effect?

Reply:
yes